yarn.lock

```diff
@@ -2771,6 +2771,38 @@ aws-sdk@^2.1211.0, aws-sdk@^2.596.0, aws-sdk@^2.928.0:
     uuid "8.0.0"
     xml2js "0.4.19"
 
+aws-sdk@^2.1282.0:
```

Comment:
Unless there is a specific feature in this version that you need, updates to yarn.lock shouldn't be committed.

Reply:
Reverted